packages/components/src/DateTimePickers/DateTimePicker/DateTimePicker.component.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import theme from './DateTimePicker.scss';
+import PickerHeader from '../PickerHeader';
+import DateTimeWrapper from '../DateTimeWrapper';
+import MonthYearWrapper from '../MonthYearWrapper';
+
+function DateTimePicker(props) {
+	const isCalendarView = false;
+	const viewComponent = isCalendarView
+		? <DateTimeWrapper />
+		: <MonthYearWrapper />;
+
+	return (
+		<div className={theme.container}>
+			<div className={theme.header}>
+				<PickerHeader />
```

Comment:
This should be in the DayTimeWrapper. It's only on DayPicking that we have this component.

Plus this top component really only switch from 1 implem or the other.